src/extension.ts

```diff
@@ -37,6 +39,18 @@ export function activate(context: vscode.ExtensionContext) {
     const langIntellisenseProvider = new LangIntellisenseProvider(context, remoteFileSystemProvider);
     context.subscriptions.push( ...langIntellisenseProvider.triggers );
 
+    // Register: [intellisense] TexSymbolProvider
```

Comment:
Don't register it in the main file. Register it together with other triggers in `langIntellisenseProvider.ts`